### PR TITLE
README.md: typo

### PR DESCRIPTION
**Pull request by RubenKelevra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ $ sudo pacman -S go-ipfs
 
 Development version of go-ipfs is also on AUR under
 [go-ipfs-git](https://aur.archlinux.org/packages/go-ipfs-git/).
-You can install it using your favourite AUR Helper or manually from AUR.
+You can install it using your favorite AUR Helper or manually from AUR.
```
Comment (Stebalien) [inline README.md]: This is the UK spelling.

File: README.md [hunk 2]
```diff
@@ -302,7 +302,7 @@ SUBCOMMANDS
 
   ADVANCED COMMANDS
     daemon        Start a long-running daemon process
-    mount         Mount an ipfs read-only mountpoint
+    mount         Mount an ipfs read-only mount point
```
Comment (Stebalien) [inline README.md]: This is copied from `ipfs --help`.

File: README.md [hunk 3]
```diff
@@ -352,7 +352,7 @@ Watch the ipfs log:
 
 Wait for ipfs to start. ipfs is running when you see:
 
-    Gateway (readonly) server
+    Gateway (read-only) server
```
Comment (Stebalien) [inline README.md]: This is command output (go-ipfs/cmd/ipfs/daemon.go:626)

**Comment by RubenKelevra** [comment]
> Thanks! We need to update the relevant commands first.

fixed :)

**Comment by Stebalien** [comment]
Looks like the tests need to be updated to match the new text: https://app.circleci.com/pipelines/github/ipfs/go-ipfs/2277/workflows/0f67de07-cc03-49a3-b1f8-ad7beb2d5ec9/jobs/28027/tests

**Pull request merged**